pyaedt/modeler/Primitives.py

```diff
@@ -1421,9 +1449,19 @@ def refresh(self):
         self.add_new_objects()
 
     def cleanup_objects(self):
-        """Clean up any objects in self.objects that have been removed by previous operations
-        and do not exist in the modeler anymore. Also updates object ids which may have changed
-        via a modeler operation such as unite"""
+        """Clean up objects that no longer exist in the modeler because 
+        they were removed by previous operations.
+        
+        This method also updates object IDs that may have changed via  
+        a modeler operation such as :funct:`pyaedt.modeler.Model3D.Modeler3D.unite` 
+        or :funct:`pyaedt.modeler.Model2D.Modeler2D.unite`.
```

Comment:
```suggestion
        a modeler operation such as :func:`pyaedt.modeler.Model3D.Modeler3D.unite` 
        or :func:`pyaedt.modeler.Model2D.Modeler2D.unite`.
```
I think the directive is :func, right?